include/materials/ce_nuclide.hpp

```diff
@@ -47,6 +47,7 @@ class CENuclide : public Nuclide {
   double nu_delayed(double E_in, std::size_t i) const override final;
   double reaction_xs(uint32_t mt, double E_in, size_t i) const override final;
   double elastic_xs(double E_in, std::size_t i) const override final;
+  double heating_xs(double E_in, std::size_t i) const override final;
```

Comment:
```suggestion
  double heating(double E_in, std::size_t i) const override final;
```
I think we should just call this heating, as it isn't really a cross section.